joy_node:
  ros__parameters:
    device_id: 0
    deadzone: 0.05
    autorepeat_rate: 20.0

teleop_node:
  ros__parameters:
    axis_linear:
      x: 1 # CHANGE
    scale_linear:
      x: 0.3
    scale_linear_turbo:
      x: 0.5
    
    #axis_linear:
    #  y: 3 # CHANGE
    #scale_linear:
    #  y: 0.3
    #scale_linear_turbo:  
    #  y: 0.5

    axis_angular:
      yaw: 0 # CHANGE
    scale_angular:
      yaw: 0.3
    scale_angular_turbo:  
      yaw: 0.5
    

    enable_button: 6
    enable_turbo_button: 7

    require_enable_button: true

linear_position_pub:
  ros__parameters:
    min_value: 0.0
    max_value: 0.10
    axis_index_x: 3
    axis_index_y: 4

    
    
